src/main.rs

```diff
@@ -152,8 +153,12 @@ fn cli() -> clap::Command {
             Command::new("generate-stub")
                 .alias("gen")
                 .about("Generate input handling code for a given language")
-                .arg(arg!(<PROGRAMMING_LANGUAGE> "programming language of the solution stub"))
-                .arg(arg!(--"debug" "generate the stub for the reference generator"))
+                .arg(arg!(<PROGRAMMING_LANGUAGE> "Programming language of the solution stub"))
+                .arg(
+                    arg!(--"from-file" <STUBFILE> "Generate stub from stubgen file instead of the current clash")
+                        .value_parser(clap::value_parser!(PathBuf))
+                )
+                .arg(arg!(--"from-reference" "Generate stub from a reference stub generator instead of the current clash").conflicts_with("from-file"))
```

Comment:
The point about "the" vs "a" is valid, I changed it to "the" since there is only really the one reference stub generator. I wouldn't change the name though because I think the name should mirror the `--from-file` arg which does a very similar thing – replaces the stubgen of the current clash with something else.